src/transformers/models/glpn/convert_glpn_to_pytorch.py

```diff
@@ -132,11 +132,11 @@ def convert_glpn_checkpoint(checkpoint_path, pytorch_dump_folder_path, push_to_h
     config = GLPNConfig(hidden_sizes=[64, 128, 320, 512], decoder_hidden_size=64, depths=[3, 8, 27, 3])
 
     # load feature extractor (only resize + rescale)
```

Comment:
I got lucky today:

```
load image processor
```

I am going to buy loto

Reply:
:D You're more thorough than I! Can you tell I didn't search for "feature extractor" with a space?